services/network/WebSocketServerConnectionObserver.cpp

```diff
@@ -55,7 +55,7 @@ namespace services
 
     std::size_t WebSocketServerConnectionObserver::MaxSendStreamSize() const
     {
-        return sendBuffer.max_size();
+        return std::min(sendBuffer.max_size(), Subject().MaxSendStreamSize() - 4);
```

Comment:
Where does the minus four come from?

Reply:
4 isthe maximum frame header size; code is updated